io/module_wrt_grid_comp.F90

```diff
@@ -1896,6 +1776,23 @@ subroutine wrt_run(wrt_comp, imp_state_write, exp_state_write,clock,rc)
                            fieldbundle=file_bundle, rc=rc)
         if (ESMF_LogFoundError(rcToCheck=rc, msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
 
+#if 1
```

Comment:
this #if 1 seems like some testing section? Do you need this here?

Reply:
Removed